docker-compose.yaml

```diff
@@ -1,68 +1,106 @@
 version: '3.5'
 
+x-base:
+  &default-base
+  build:
+    context: .
+    dockerfile: Dockerfile
+  image: ssvnode:latest
+  command: make BUILD_PATH=/go/bin/ssvnode start-node
+  networks:
+    - bloxapp-docker
+  restart: always
+
+x-base-dev:
+  &default-dev
+  << : *default-base
+  image: ssvnode-debug:latest
+  build:
+    context: .
+    dockerfile: dev.Dockerfile
+  command:  air
+  security_opt:
+    - "seccomp:unconfined"
+  cap_add:
+    - SYS_PTRACE
+  volumes:
+    - ./:/go/src/github.com/bloxapp/ssv
+
+
 services:
   ssv-node-1:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    command:
-      - "start-node"
-      - "--node-id=1"
-      - "--leader-id=1"
-      - "--private-key=${SSV_NODE_1}"
-      - "--validator-key=${VALIDATOR_PUBLIC_KEY}"
-      - "--beacon-node-addr=${BEACON_NODE_ADDR}"
-      - "--network=${NETWORK}"
-      - "--val=${CONSENSUS_TYPE}"
+    <<: *default-base
+    container_name: ssv-node-1
     environment:
       DOCKER_COMPOSE: "true"
+      NODE_ID: 1
+      SSV_PRIVATE_KEY: ${SSV_NODE_1}
       PUBKEY_NODE_2: "98090f5ace3818b27b6ea2d38524556ec8c39d568a1595adb00964415eae62998c12ef015eda17f8d590f22b1778ec02"
```

Comment:
**Out-of-Scope but as a suggestion:** maybe we could parameterized the keys instead of hard-coding them here?

Reply:
already did